README.md

```diff
@@ -101,6 +102,7 @@ $ source ~/.bash_profile
 ```
 setx TF_VAR_tenancy_ocid <value>
 setx TF_VAR_user_ocid <value>
+setx TF_VAR_compartment_ocid <value>
```

Comment:
We also mention setting the compartment on line 114. That line could be removed with this change... although actually I see that the compute/instance example we're telling users to run has other variables that must be set for it to work. So that's really not the right example to point them to.

Really, I think a bigger cleanup is needed here. For one, the other variables listed here are all ones that are required for the provider - that's not true of compartment_ocid. However, these variable names are also just a convention that we use in our examples - there's nothing magic about them. We do though, have magic variable names that just work with the provider, see https://github.com/oracle/terraform-provider-oci/blob/master/provider.go#L53, and we're not documenting that anywhere.

Reply:
Until we do a bigger change around this, I think I'll recommend keeping what you have, switching the example we reference to networking/vcn (it's very simple example that only needs the variables mentioned here, I think, should verify) and remove line 114.

Reply:
I'll take your suggestions. From a test perspective, the compartment_ocid is a defacto required variable; which is why I decided to treat it as such. Perhaps as part of the bigger clean-up we may need to add setup steps that are relevant to running the tests.

Reply:
@briangustafson I've addressed your feedback in the latest commit